src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -69,7 +88,10 @@ public function __construct(
         CheckoutShippingStep $checkoutShippingStep,
         CheckoutPaymentStep $checkoutPaymentStep,
         CheckoutFinalizeStep $checkoutFinalizeStep,
-        CheckoutThankYouPage $checkoutThankYouPage
+        CheckoutThankYouPage $checkoutThankYouPage,
+        PaypalPage $paypalPage,
```

Comment:
We should be a bit more specific here, PayPalCheckoutExpressPage? There are other payment gateways from PayPal that we will handle in near future.